src/tree/AzureDBAPIStep.ts

```diff
@@ -29,6 +29,18 @@ export class AzureDBAPIStep extends AzureWizardPromptStep<IPostgresServerWizardC
             placeHolder: localize('selectDBServerMsg', 'Select an Azure Database Server.')
         });
 
+        if (result.data?.api !== API.Postgres) {
+            const modelPicks: IAzureQuickPickItem<CapacityModelName>[] = [
+                {label: "Provisioned Throughput", data: "Provisioned"},
+                {label: "Serverless", data: "Serverless"}
+            ];
+            const modelResult: IAzureQuickPickItem<CapacityModelName> = await ext.ui.showQuickPick(modelPicks, {
```

Comment:
You need to create a separate `AzureWizardPromptStep` for this otherwise the back button won't work properly. Each call to `showQuickPick` or `showInputBox` should be it's own step